testsuite/unit/TAP.lua

```diff
@@ -0,0 +1,104 @@
+-- This is TAP.lua from https://github.com/lunarmodules/busted/tree/master/busted/outputHandlers
```

Comment:
If this output handler is shipped with busted, you shouldn't have to copy it to this project. Once you have the right `LUAPATH` set, you should be able to use it from the installed busted scripts.

Do you like the TAP output better than the standard text output? If it looks alright in the build, we could use it to replace the text output handler in `testsuite/unit/output.lua`

Reply:
Yep.  I can just have "--output TAP".   Cool.